### PR TITLE
Migrate Button component to tailwind (#232)

### PR DESCRIPTION
**Pull request by kakaly**
# Details

Migrated Button component to tailwind

## Description

* Defined new tailwind classes inside tailwind.css

* Defined new colors inside tailwind.js

* Modified Button to use the newly defined css classes

### CODE REVIEW

**Comment by crysfel** [comment]
Thanks for contributing, I left a couple comments. Good Job!

File: src/theme/tailwind.css [hunk 1]
```diff
@@ -1,7 +1,135 @@
 @tailwind preflight;
-@tailwind utilities;
 @tailwind components;
 
+/* button */
+.button {
```
[the remark this answers is not in the document]
Comment (kakaly) [reply src/theme/tailwind.css]: Sounds good. Updated the PR accordingly. We can merge it if you think it looks good.

**Comment by emmawedekind** [comment]
Hello - what's the status on this!

**Comment by kakaly** [comment]
@emmawedekind - I have addressed @crysfel 's comments and updated the changes already. Let me know if you have any comments on this. Otherwise, it is good to merge.

**Comment by tureey** [comment]
Everything what was asked has been implemented as I can see. Ready to merge IMHO but wait for more people if so

File: tailwind.js [hunk 2]
```diff
@@ -54,21 +53,22 @@ let colors = {
   'primary-lightest': '#D5FFF4',
   'primary-lighter': '#E6F5F0',
   'primary-light': '#69D5B1',
-  'primary': '#00bc89',
+  'primary-blur': '#D5FFF4',
```
Comment (crysfel) [inline tailwind.js]: Please remove `primary-blur` and use `primary-lightest` as is the same value.

Comment (kakaly) [reply tailwind.js]: Done!

**Comment by crysfel** [comment]
Build is failing on Travis, can you take a look?

**Comment by kakaly** [comment]
@crysfel - fixed it. I modified danger-darker to danger-dark when i resolved merge conflicts by mistake. Reverted it to danger-darker. Should be okay now.

**Pull request merged**